initramfs-init.in

```diff
@@ -363,6 +363,11 @@ if [ -n "$KOPT_nbd" ]; then
 	setup_nbd || echo "Failed to setup nbd device."
 fi
 
+if [ "$KOPT_rootfstype" = "zfs" ]; then
+	# zpool reports /dev/zfs missing if it can't read /etc/mtab
+	ln -s /proc/mdstat /etc/mtab
```

Comment:
shouldn't the symlink point to `/proc/mounts`? In any case, it sounds like an upstream bug.